HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -2137,101 +2145,126 @@ def self.apply_hvac_equipment_adjustments(mj, runner, hvac_sizings, weather, hva
       end
 
       cool_cap_rated = hvac_sizings.Cool_Load_Tot / total_cap_curve_value
+      # cool_cap_rated = hvac_sizings.Cool_Load_Tot / (total_cap_curve_value*total_cap_flow_mod_fac), in E+ Eng Reference
 
+      cool_cfm_rated = UnitConversions.convert(cool_cap_rated, 'btu/hr', 'ton') * hvac_cooling_ap.cool_rated_cfm_per_ton[hvac_cooling_speed]
+
+      # rated SHR
       hvac_cooling_shr = hvac_cooling_ap.cool_rated_shrs_gross[hvac_cooling_speed]
-      sens_cap_rated = cool_cap_rated * hvac_cooling_shr
 
-      # Calculate the air flow rate required for design conditions
+      rated_barometric_pressure_psi = UnitConversions.convert(1, 'atm', 'psi') # assume rated pressure is at sea level
+
+      hr_indoor_cooling_rated = Psychrometrics.w_fT_Twb_P(HVAC::AirSourceCoolRatedIDB, HVAC::AirSourceCoolRatedIWB, rated_barometric_pressure_psi) # [lbm/lbm]
+
+      # calculate A_o using rated conditions
+      # Don't use coil ao factor in hvac.rb to calculate design SHR. That ao has specific use case of finding rated SHRs for variable speed equipment.
+      a_o_rated = Psychrometrics.CoilAoFactor(HVAC::AirSourceCoolRatedIDB, rated_barometric_pressure_psi, UnitConversions.convert(cool_cap_rated, 'btu/hr', 'kbtu/hr'), cool_cfm_rated, hvac_cooling_shr, hr_indoor_cooling_rated)
+      cool_cap_design = hvac_sizings.Cool_Load_Tot
```

Comment:
@Sashadf1 Maybe add some comments above this line indicating this is an initial assumption prior to attempting to balance sensible and latent design capacity by applying the over size limits.

Reply:
ok, will do.

Can you clarify the line number, and which initial assumption?

Reply:
I was thinking the line `cool_cap_design = hvac_sizings.Cool_Load_Tot` could use an explanation. I know it's obvious based the code that comes next but I thought additional comments would be worth it.